CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,197 @@
+* Deliberate intimidation or stalking (online or in person).
+* Sustained disruption of community spaces or events. This includes bringing
+  weaponry into community spaces; see below for a more precise definition.
+* Intentional violation of local laws or regulations at in-person events.
+* Advocating for, encouraging, or assisting in any of the above behaviour.
+
+'Weapons' include, but are not limited to, guns, explosives (including 
+fireworks without prior approval by a community organizer), and large knives 
+such as those used for hunting or display, as well as any other item used for 
+the purpose of causing injury and harm to others. 
+
+## Consequences of unacceptable behaviour
+
+Unacceptable behaviour from any community member, including anyone with
+decision-making authority, will not be tolerated. Anyone asked to stop
+unacceptable behaviour is expected to comply immediately.
+
+If a community member engages in unacceptable behaviour, the community
+organizers may take any action they deem appropriate, subject to correction 
+and reversal by the lead organizer. These include, but are not limited to:
+
+* A verbal warning (either in private or in public).
+* Eviction from an event with no refund (if applicable), with conditional 
+  right of return or without right of return.
+* A temporary ban of any length, with or without right of appeal, from any or
+  all community spaces.
+* A permanent ban, with or without right of appeal, from any or all community
+  spaces.
+* Temporary revocation of community organizer status for any length of time, 
+  with or without right of appeal.
+* Permanent revocation of community organizer status with or without right of 
+  appeal.
+* Removal of contributions to Pext from any Pext-associated repositories, with
+  or without right of appeal.
+* Notification of law enforcement, or other appropriate authorities, along with
+  any evidence of wrongdoing provided to the community organizers.
+
+## Reporting guidelines
+
+If you are subject to unacceptable behaviour, witness unacceptable behaviour in
+progress, or have any other
+concerns, please notify a community organizer as soon as possible. The current
+community organizers are:
+
+* Sylvia van Os (sylvia@hackerchick.me) (lead organizer)
+* Emily Lau (fuchslein@hackerchick.me)
+
+When reporting, please include the following information, to the best of your
+knowledge and ability:
+
+* When and where the unacceptable behaviour occurred.
+* Who engaged in unacceptable behaviour.
+* Which unacceptable behaviour(s) happened.
+* Any evidence for your report (such as screencaps, logs, etc).
+* Who (besides yourself) was affected by this behaviour, if applicable.
+* Whether you want any additional actions taken by the community organizers to 
+  ensure that your safety and well-being remain intact.
+
+Community organizers can assist local authorities and connect you
+with help or providing a safe environment provided they are able. In the 
```

Comment:
I cannot parse this sentence anymore, afraid this one is going to need a rephrase for readability purposes. I'm pretty sure this is the last ever thing I'll have to ask.

Reply:
How about "Provided they are able, community organizers can assist local authorities, connect you with help or provide a safe environment."? I think that's what you meant to say here?